highfidelity/hifi
# Fix audio qt59

Test plan
- build with Qt 5.9
- open Tablet
- click Audio button
- make sure there is a list of audio devices available
